/ls1550_left/lslidar_ls_driver_node:
  ros__parameters:
    device_ip: 192.168.1.200
    msop_port: 2368
    difop_port: 2369
    frame_id: laser_link
    add_multicast: false
    group_ip: 224.1.1.2
    time_synchronization: false
    min_range: 0.3
    max_range: 500.0
    scan_start_angle: -60
    scan_end_angle: 60
    topic_name: lslidar_point_cloud
    packet_loss: false        #是否开启丢包检测
    packet_rate: 10000.0
    #pcap: /home/ls/work/fuhong/data/ls128/20221012-ntp.pcap                        #pcap包路径，加载pcap包时打开此注释
